scripts/package_ota.py

```diff
@@ -59,19 +97,36 @@ def process_file(fn, name, full_check=True, has_ab=True):
     }
     shutil.copy(fn, OTA_OUTPUT_DIR / f"{fn.stem}-{hash_raw}.img")
 
+  if gpt:
+    ret["gpt"] = {
+      "lun": lun,
+      "start_sector": start_sector,
+      "num_sectors": num_sectors,
+    }
+
   return ret
 
 
 if __name__ == "__main__":
+  parser = argparse.ArgumentParser(description='Generate a partitions manifest')
+  parser.add_argument('--all-partitions', action='store_true', help='include all partitions, not only the OTA ones')
+  args = parser.parse_args()
```

Comment:
this should always generate both manifests together so we always have a matching set